app/components/Account/AccountVoting.jsx

```diff
@@ -925,6 +951,11 @@ class AccountVoting extends React.Component {
                                         proxy={this.state.proxy_account_id}
                                     />
                                 </div>
+                                <JoinWitnessesModal
+                                    show={this.state.showCreateWitnessModal}
```

Comment:
Stick to common wording and usage of modals. (`visible` e.g.)

Reply:
ok